plugins/anonAuthPlugin.js

```diff
@@ -2,6 +2,7 @@ import Boom from 'boom';
 import chalk from 'chalk';
 import Rx from 'rxjs';
 
+const AUTH_TIMEOUT = 5000;
```

Comment:
Actually can we use a configuration for this value? So it's defined in fewer spots.